src/styles/card__styles/src/styles/blocks/stars.scss

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    width: $star-size;
+    height: $star-size;
+    background-repeat: no-repeat;
+    background-position: center;
```

u can use just `background` for it 
https://developer.mozilla.org/en-US/docs/Web/CSS/background
```suggestion
    width: $star-size;
    height: $star-size;
```